lib/SPIRV/SPIRVReader.cpp

```diff
@@ -2808,6 +2808,14 @@ void generateIntelFPGAAnnotation(const SPIRVEntry *E,
       Out << ":" << Str;
     Out << '}';
   }
+  if (E->hasDecorate(DecorationBankBitsINTEL)) {
+    Out << "{bank_bits:";
+    auto Literals = E->getDecorationLiterals(DecorationBankBitsINTEL);
+    for (size_t I = 0; I < Literals.size() - 1; ++I) {
```

Comment:
nit: no need for brackets here and after for single-string loops, but if the rest code in the file is also written in the same way, I'm OK with leaving it as is.